### ISSUE TITLE
[dotfiles.sh] Incorrect resolution of dotfiles/dotfiles-manager repo paths

### ISSUE THREAD

**Issue by rucker** [rucker/dotfiles#30]
Scenario: Invoking `dotfiles.sh --install` from an external PowerShell script. In this case the calling script is located at `~/repos/config/Win/setup.ps1` 

Issue: Incorrect paths to repos observed when:
- `dotfiles.sh` clones dotfiles-manager (it is placed in parent dir of calling PowerShell script)
- Symlinks to `dotfiles.sh` and `dfm.py` are created

Console output:
```
C:\Users\IEUser\repos\config\Win [master ≡ +1 ~2 -0 !]> .\setup.ps1
Installing...
Cloning Dotfiles Manager
Cloning into 'dotfiles-manager'...
remote: Enumerating objects: 1378, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (15/15), done.
remote: Total 1378 (delta 6), reused 9 (delta 3), pack-reused 1360
Receiving objects: 100% (1378/1378), 218.50 KiB | 2.04 MiB/s, done.
Resolving deltas: 100% (826/826), done.
Creating /c/Users/IEUser/bin
Linking /c/Users/IEUser/bin/dfm -> /c/Users/IEUser/repos/config/dotfiles-manager/dotfilesmanager/dfm.py
Linking /c/Users/IEUser/bin/dotfiles -> /c/Users/IEUser/repos/config/dotfiles/dotfiles.sh
ln: failed to create symbolic link '/c/Users/IEUser/bin/dotfiles': No such file or directory
C:\Users\IEUser\repos\config\Win [master ≡ +1 ~2 -0 !]> ls ..


    Directory: C:\Users\IEUser\repos\config


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
<snip>
d-----        11/4/2021   6:23 PM                dotfiles-manager
<snip>
```

**Comment by rucker** [comment]
Additional info: This occurs even when invoking `dotfiles.sh` from an absolute path from the CLI (bash). One issue here is that the [line in question](https://github.com/rucker/dotfiles/blob/5a995ebf31500c8a23715aba027e12eb2c468b04/dotfiles.sh#L82) omits the `$` when referencing `$BASH_SOURCE[0]`.

**Comment by rucker** [comment]
This appears to be resolved as of d267886
